components/x-audio/src/components/styles.scss

```diff
@@ -359,3 +358,36 @@ $primary-scrubber-color: oColorsGetPaletteColor('teal');
 // 	width: var(--buffered-width);
 // 	background-color: green;
 // }
```

Comment:
Oh I know this change is nothing to do with this PR but I think this code was accidentally committed. That's right isn't it @robsquires? I think this was an experiment for buffering stuff that we're not doing anymore.

Reply:
yup totally, good spot!